### DISCUSSION TITLE
spring-boot-2/5-5-redis-publisher-subscriber

### DISCUSSION THREAD

**Discussion by giscus** [dyc87112/giscus-didispace#23] [Announcements]
# spring-boot-2/5-5-redis-publisher-subscriber

Spring Boot 2.x基础教程：使用Redis的发布订阅功能 通过前面一篇集中式缓存的使用教程，我们已经了解了Redis的核心功能：作为K、V存储的高性能缓存。 接下来我们会分几篇来继续讲讲Redis的一些其他强大用法！如果你对此感兴趣，一定要关注收藏我哦！ 发布订阅模式 如果你看过之前我写的关于MQ的相关文章，那么对于发布订阅功能应该不会陌生。如果没看过，那也不要紧，这里先做一个简单介绍，已经了解的可以跳过直接看下一节内容。

https://www.didispace.com/spring-boot-2/5-5-redis-publisher-subscriber.html

**Comment by YU-7** [comment]
勘误：“这里为了简单实现，公用CHANNEL名称字段，我都写在了应用主类里”，公用 -> 共用
